assets/styles.scss

```diff
@@ -71,3 +71,79 @@ html.has-custom-scrollbars {
     border-color: transparent;
   }
 }
+
+html.has-light-theme {
+  background-color: #F8F8F8;
+
+  body {
+    color: #333;
+  }
+
+  h1.title {
+    text-shadow: 1px 0px #ffdd57;
+  }
```

Comment:
These probably make sense here since they are also defined in this file. You can make it more `scss` like and use `&`.